metaversefile-api.js

```diff
@@ -480,6 +481,9 @@ metaversefile.setApi({
       },
     };
   },
+  useIO() {
+    return ioManager;

```

Comment:
IMO apps should not be touching `ioManager`, that is a major can of worms.

If we need a new control mode then that can be handled here in `app` by setting the mode, rather than giving raw control to apps.